samples/RazorSlices.Samples.WebApp/Program.cs

```diff
@@ -1,8 +1,12 @@
 using RazorSlices;
 using RazorSlices.Samples.WebApp;
+using RazorSlices.Samples.WebApp.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddSingleton<LoremService>();
+builder.Services.AddHttpContextAccessor();
```

Comment:
Using `IHttpContextAccessor` is a bit of an anti-pattern. Can you log an issue to support `HttpContext` on slices separately when used in the context of a request (e.g. on `RazorSliceHttpResult`) rather than use it as an example for DI please.

Reply:
I was not aware that using `IHttpContextAccessor` can be considered an anti-pattern. I have submitted an issue #22 to request support for `HttpContext` on slices. Thank you for pointing this out 🚀